DigitalLearningSolutions.Data/Models/CustomPrompts/CourseAdminFieldsResult.cs

```diff
@@ -1,23 +1,19 @@
-﻿namespace DigitalLearningSolutions.Data.Models.CustomPrompts

```

Comment:
Are the line endings correct in this file?

Reply:
As far as I can tell I have CRLF selected in Rider, how can you tell the line endings are incorrect?

Reply:
if you open the file in any decent text editor (e.g. rider, visual studio, visual studio code. notepad++) it'll tell you in some way. (see the screenshot in the main review message for rider). You can tell in pull requests as they show up as lots of changes, but there is no actual content changes between them.

Reply:
Ohhh I think it's showing up as lots of changes because I renamed the file. In Rider it always says CRLF for me on the bottom toolbar.